core/src/main/java/org/apache/accumulo/core/client/TableOfflineException.java

```diff
@@ -26,7 +27,7 @@ private static String getTableName(Instance instance, String tableId) {
     if (tableId == null)
       return " <unknown table> ";
     try {
-      String tableName = Tables.getTableName(instance, tableId);
+      String tableName = Tables.getTableName(instance, new Table.ID(tableId));
```

Comment:
HBase does this with their `TableName` class to avoid tons of objects hanging around needing GC. The API is pretty nice as an end-user:

```java
Connection conn = ...;
Table t = conn.getTable(TableName.valueOf("josh_table1"));
```